ui/lib/core/addon/components/form-field.js

```diff
@@ -42,6 +42,23 @@ import { dasherize } from 'vault/helpers/dasherize';
  */
 
 export default class FormFieldComponent extends Component {
+  // alphabetical list of all editTypes if they hide <FormFieldLabel>
+  shouldHideLabel = {
```

Comment:
Curious about feedback here. Having this object felt like a nice, concise way to document the list of different edit types and quickly reference whether or not they render a `<FormFieldLabel>`. 

But if this feels overly complicated I can refactor to more similarly mimic the template's original conditional